hm_pyhelper/miner_json_rpc/client.py

```diff
@@ -0,0 +1,26 @@
+from jsonrpcclient import request
+
+
+class Client(object):
+
+    def __init__(self, url='http://localhost:4467'):
+        self.url = url
+
+    def __fetch_data(self, method, **kwargs):
+        response = request(self.url, method, **kwargs)
```

Comment:
How are failed requests handled?

Reply:
We let the exception bubble up to the person using the library to handle it how they see fit, either they can let it bubble up to 500 their app (in the case of hm-diag) or they can catch it and handle accordingly.